Gemfile

```diff
@@ -1,2 +1,3 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
\ No newline at end of file
+gem 'github-pages', group: :jekyll_plugins
+gem "webrick", "~> 1.7"
```

Comment:
why is this module needed?

Reply:
just for running my code locally. i can remove it if you want

Reply:
yes please

Reply:
done!